loops.js

```diff
@@ -7,6 +7,15 @@
 // ****
 // *****
 
+
+for (let i = "*"; i < "*****"; i++) {
+    console.log(i)
+    console.log(i.repeat(2))
+    console.log(i.repeat(3))
+    console.log(i.repeat(4))
+    console.log(i.repeat(5));
+}
+
```

Comment:
dit kan ook: 

for (let i = 0; i < 5; i++) {
  console.log('*'.repeat(i + 1));
}